px-context-browser-panel.html

```diff
@@ -12,6 +12,10 @@
         <div class="loading-bar"></div>
       </template>
       <slot name="filter"></slot>
+      <template is="dom-if" if="[[favoritesCurrentlySyncing]]">
```

Comment:
Seems like we shouldn't have two loading bar elements with separate dom-ifs and properties exposed by px-context-browser-panel. Instead in px-context-browser.html we should decide if `showLoadingBar` is true if the favorites are syncing OR some other condition is true.

Reply:
👍